BuildScripts/mvComplexFunctions.sql

```diff
@@ -917,6 +917,7 @@ BEGIN
 
             tLastType               := tDmlType;
             iArraySeq               := 1;
+            uRowIDArray             := '{}';
```

Comment:
This section of code is batching the processing of log table entries into groups with the same change type. So if there are 8 `UPDATE` changes to answers, then 8 `DELETE` changes to other answers, it'll do the updates all in one batch, then the deletes all in one batch. 

The problem was when you get e.g. 8 `UPDATE` changes followed by say 4 `DELETE` changes, it'll batch the 8 updates, process them, and then batch the 4 deletes but _use the same array as before_, so the indexes that haven't been overwritten still have the last 4 updates in them. That results in those 4 getting deleted from the analytics table 🤦